### ISSUE TITLE
eyes direction 

### ISSUE THREAD

**Issue by vespersland** [neuralchen/SimSwap#311]
i don't know why eyes direction always look to the right with some sources...any suggestion?
My sources images are perfectly symmetrical (doesn't look on the side)
thanks in advance!

**Comment by andrissa** [comment]
are u using new model 512 ? if yes maybe same as me, that trained photo have a lot of people with the eyes direction on right

**Comment by vespersland** [comment]
> are u using new model 512 ? if yes maybe same as me, that trained photo have a lot of people with the eyes direction on right

Yes i use new model 512,thanks for your answer,hoping to fix this soon!
